src/slic3r/GUI/DesktopIntegrationDialog.cpp

```diff
@@ -539,7 +549,7 @@ void DesktopIntegrationDialog::perform_downloader_desktop_integration()
     std::string desktop_file_downloader = GUI::format(
         "[Desktop Entry]\n"
         "Name=PrusaSlicer URL Protocol%1%\n"
-        "Exec=\"%2%\" --single-instance %%u\n"
+        "Exec=%2% --single-instance %%u\n"
```

Comment:
@eliadevito Hi, what is the reason for this change? Will this work correctly when there are spaces in the path?

Reply:
@lukasmatena Hi, I moved quotes to line 505 to exclude them from flatpak path because they are not required.

https://github.com/prusa3d/PrusaSlicer/blob/95403492457852cf4ea7f67ca58bbc0ef54f9942/src/slic3r/GUI/DesktopIntegrationDialog.cpp#L505 

In the flatpak case the exec line must simply be composed as follow:
`flatpak run FLATPAK_ID APP_PARAMS`